lib/sync-settings.coffee

```diff
@@ -42,17 +42,23 @@ module.exports =
       "packages.json":
         content: JSON.stringify(@getPackages(), null, '\t')
       "keymap.cson":
-        content: @fileContent atom.keymaps.getUserKeymapPath()
+        content: (@fileContent atom.keymaps.getUserKeymapPath()) ? "# keymap file (not found)"
       "styles.less":
-        content: @fileContent atom.styles.getUserStyleSheetPath()
+        content: (@fileContent atom.styles.getUserStyleSheetPath()) ? "// styles file (not found)"
       "init.coffee":
-        content: @fileContent atom.config.configDirPath + "/init.coffee"
+        content: (@fileContent atom.config.configDirPath + "/init.coffee") ? "# initialization file (not found)"
       "snippets.cson":
-        content: @fileContent atom.config.configDirPath + "/snippets.cson"
+        content: (@fileContent atom.config.configDirPath + "/snippets.cson") ? "# snippets file (not found)"
 
     for file in atom.config.get('sync-settings.extraFiles') ? []
+      ext = file.slice(file.lastIndexOf(".")).toLowerCase()
+      cmtstart = "#"
+      cmtstart = "//" if ext in [".less", ".scss", ".js"]
+      cmtstart = "/*" if ext in [".css"]
+      cmtend = ""
+      cmtend = "*/" if ext in [".css"]
       files[file] =
-        content: @fileContent atom.config.configDirPath + "/#{file}"
+        content: (@fileContent atom.config.configDirPath + "/#{file}") ? "#{cmtstart} extra (not found) #{cmtend}"
```

Comment:
Can you replace `"#{cmtstart} extra (not found) #{cmtend}"` with `"#{cmtstart} {file} file (not found) #{cmtend}"`.